src/main/java/io/managed/services/test/cli/CLI.java

```diff
@@ -284,23 +297,35 @@ public void grantAdminAcl(ACLEntityType aclEntityType, String entityIdentificato
      * Return the Registry in use from the CLI
      */
     public Registry createServiceRegistry(String name) throws CliGenericException {
-        return retry(() -> exec("service-registry", "create", "--name", name))
-                .asJson(Registry.class);
+        var process = retry(() -> exec("service-registry", "create", "--name", name));
```

Comment:
I already made most of this refactoring here: https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/tree/502-1 , Just running it, in case it will pass i will commit it here.

Reply:
in a nutshell: 
- refactor service registries: 
removal of registryclient2, 
unnecessary passing of offline token to classes implementing BaseApiClient.